scripts/scil_tractogram_resample.py

```diff
@@ -3,28 +3,34 @@
 
 """
 Script to resample a tractogram to a set number of streamlines.
+
 Default behavior:
-- IF number of requested streamlines is lower than streamline count: DOWNSAMPLE
-- IF number of requested streamlines is higher than streamline count: UPSAMPLE
-To prevent upsample if not desired use --never_upsample.
+- If the number of requested streamlines is lower than the streamline count:
+DOWNSAMPLE
+- If the number of requested streamlines is higher than the streamline count:
+UPSAMPLE  (to prevent upsampling when not desired, use --never_upsample).
 
-Can be useful to build training sets for machine learning algorithms, to
-upsample under-represented bundles or downsample over-represented bundles.
+Upsampling:
+    Works by generating new streamlines, either by adding gaussian noise to
```

Comment:
misses a "or" to that "either"

Reply:
Specifically, the "or" is generating streamlines via parallel transport.